inc/class-s3-uploads-stream-wrapper.php

```diff
@@ -638,11 +638,15 @@ public function rename($path_from, $path_to)
 				isset($options['acl']) ? $options['acl'] : 'private',
 				$options
 			);
-			// Delete the original object
-			$this->getClient()->deleteObject([
-				'Bucket' => $partsFrom['Bucket'],
-				'Key'    => $partsFrom['Key']
-			] + $options);
+			try {
+				// Delete the original object
+				$this->getClient()->deleteObject([
+					'Bucket' => $partsFrom['Bucket'],
+					'Key'    => $partsFrom['Key']
+				] + $options);
+			} catch (Exception $e) {
+
```

Comment:
Could you add a call to `$this->triggerError` here with the exception message? That seems like the best way to surface this.